<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBarFigureWidget</class>
 <widget class="QWidget" name="ToolBarFigureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>369</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="polygonButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../빈폴리곤.png</normaloff>../빈폴리곤.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="lineButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>4</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../선.png</normaloff>../선.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="circleButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../빈원.png</normaloff>../빈원.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="rectangleButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../빈사각형.png</normaloff>../빈사각형.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="triangleButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../빈삼각형.png</normaloff>../빈삼각형.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="rhombusButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../빈마름모.png</normaloff>../빈마름모.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="parallelogramButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../빈평행사변형.png</normaloff>../빈평행사변형.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="cursorButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../커서모드.png</normaloff>../커서모드.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
